src/main/webapp/app/shared/components/molecules/onboarding-dialog/onboarding-dialog.html

```diff
@@ -0,0 +1,13 @@
+<div class="dialog-body">
+  <div class="content">
+    <p jhiTranslate="onboarding.body"></p>
+    <p-message severity="warn">
+      <p jhiTranslate="onboarding.disclaimer"></p>
+    </p-message>
+  </div>
+
+  <div class="actions">
+    <jhi-button (click)="markOnboarded(false)" [label]="'onboarding.actions.imApplicant'" [shouldTranslate]="true" variant="outlined" />
+    <jhi-button (click)="markOnboarded()" [label]="'onboarding.actions.sendEmail'" [shouldTranslate]="true" />
```

Comment:
```suggestion
    <jhi-button (click)="markOnboarded(false)" [label]="onboarding.actions.imApplicant" [shouldTranslate]="true" variant="outlined" />
    <jhi-button (click)="markOnboarded()" [label]="onboarding.actions.sendEmail" [shouldTranslate]="true" />
```
The double quotation marks are not necessary here